app.drey.Warp.yaml

```diff
@@ -0,0 +1,52 @@
+app-id: app.drey.Warp
```

Comment:
the domain `drey.app` doesn't seem to exist, so this make the app-id fail to pass requirements.

Reply:
It does exist. See [app.drey](https://gitlab.gnome.org/sophie-h/app.drey/)

Reply:
it surely doesn't resolve in the browser. Also you link to a gitlab repository, so no.

See
https://docs.flatpak.org/en/latest/conventions.html#application-ids